tests/unittests/evaluation_tests.py

```diff
@@ -1083,11 +1083,16 @@ def _make_detections_dataset(self):
             ),
             predictions=fo.Detections(
                 detections=[
+                    fo.Detection(
```

Comment:
These updated unit tests fail on `develop` due to the "missing false positive" scenario, but now they pass.